spaceship/lib/spaceship/api/client.rb

```diff
@@ -0,0 +1,141 @@
+require_relative '../client'
+
+module Spaceship
+  module API
+    class Client < Spaceship::Client
+      ##
+      # Spaceship HTTP client for the App Store Connect API.
+      #
+      # This client is solely responsible for the making HTTP requests and
+      # parsing their responses. Parameters should be either named parameters, or
+      # for large request data bodies, pass in anything that can resond to
+      # `to_json`.
+      #
+      # Each request method should validate the required parameters. A required parameter is one that would result in 400-range response if it is not supplied.
+      # Each request method should make only one request. For more high-level logic, put code in the data models.
+
+      def self.hostname
+        'https://appstoreconnect.apple.com/iris/v1/'
```

Comment:
Question: is there a benefit to having a class method for this rather than a constant?

```ruby
class Client < Spaceship::Client
  HOSTNAME = 'https://appstoreconnect.apple.com/iris/v1/'

  # ...
end
```

Reply:
Nevermind, I didn't realize it was part of the interface. My apologies 😳 

___


```ruby
    # https://github.com/fastlane/fastlane/blob/master/spaceship/lib/spaceship/client.rb#L57
    def self.hostname
      raise "You must implement self.hostname"
    end
```